src/mraa.c

```diff
@@ -1130,6 +1140,32 @@ mraa_link_targets(const char* filename, const char* targetname)
     }
 }
 
+mraa_result_t
+mraa_find_uart_bus_pci(const char* pci_dev_path, char** dev_name)
+{
+    char path[PATH_MAX];
+    int max_allowable_len = 16;
```

Comment:
I'd suggest to use the same `PATH_MAX`, or at least define and use the `MAX_SIZE` we use everywhere else for path-related string limits, for consistency. If you're keen on using the current one, let's at least add `const` to denote we're using it as such.

Reply:
Using const in this case.